include/aquamarine/backend/Backend.hpp

```diff
@@ -77,7 +77,10 @@ namespace Aquamarine {
         virtual std::vector<SDRMFormat>                                 getCursorFormats()                         = 0;
         virtual bool                                                    createOutput(const std::string& name = "") = 0; // "" means auto
         virtual Hyprutils::Memory::CSharedPointer<IAllocator>           preferredAllocator()                       = 0;
-        virtual std::vector<SDRMFormat>                                 getRenderableFormats(); // empty = use getRenderFormats
+        virtual Hyprutils::Memory::CSharedPointer<IAllocator>           fallbackAllocator() {
```

Comment:
no, no, no. The dumb allocator doesn't really hold any big allocs, so it should be per-drm-backend (can be always present, no big deal)

Reply:
Don't really understand what needs to be done here. How HL should get access to dumb allocator?

Reply:
in CDRMBackend SP<CDumbAllocator>

Reply:
then it can `->output->backend`, cast that to drm and `->dumballocator`